### PR TITLE
Add Section on Env Functions 

### PR DESCRIPTION
**Pull request by idea404**
This PR adds a section explaining the environment functions in both SDKs. 

Mentioned: https://github.com/near/docs/pull/1239#issuecomment-1275813132

### CODE REVIEW

**Comment by idea404** [comment]
Hi guys, I propose to implement the table in this PR and provide examples of each of the functions following the table in a separate PR

File: docs/2.develop/contracts/environment/table-func.as.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<TableAsFunc>
+
+| Function Name          | SDK method                     | Description                                                            |
+| ---------------------- | ------------------------------ | ---------------------------------------------------------------------- |
+| SHA 256            | `context.sha256(value)`          | Hashes the random sequence of bytes using sha256.    |
+| Keccak 256        | `context.keccak256(value)`         | Hashes the random sequence of bytes using keccak256. |
```
Comment (bucanero) [inline docs/2.develop/contracts/environment/table-func.as.md:6]: I think `random` is not the right word here, as the method is hashing a user-defined `value`. (which may or may not be random data)

File: docs/2.develop/contracts/environment/table-func.as.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+| Keccak 512  | `context.keccak512(value)`   | Hashes the random sequence of bytes using keccak512. |
```
Comment (bucanero) [inline docs/2.develop/contracts/environment/table-func.as.md:7]: also here, `random`

File: docs/2.develop/contracts/environment/table-func.as.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+| Panic String       | `context.panic(message)`      | Terminates the execution of the program with the UTF-8 encoded message.  |
+| Log String      | `context.log_utf8(message)`      | Logs the string message. This message is stored on chain. |
+| Validator Stake | `context.validator_stake(account_id)` | For a given account return its current stake. If the account is not a validator, returns 0. |
+| Validator Total Stake | `context.validator_total_stake()` | Returns the total stake of validators in the current epoch. |
+| Account ID Validation | `context.isValidAccountID(account_id)` | Validates an accountId string. |
```
Comment (bucanero) [inline docs/2.develop/contracts/environment/table-func.as.md:12]:
```suggestion
| Account ID Validation | `context.isValidAccountID(account_id)` | Validates an `accountId` string. |
```

File: docs/2.develop/contracts/environment/table-func.rs.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<TableRsFunc>
+
+| Function Name          | SDK method                      | Description                                                            |
+| ---------------------- | ------------------------------- | ---------------------------------------------------------------------- |
+| SHA 256                | `env::sha256(value)` | Hashes the random sequence of bytes using sha256.                                     |
+| Keccak 256             | `env::keccak256(value)`     | Hashes the random sequence of bytes using keccak256.                                     |
+| Keccak 512             | `env::keccak512(value)`      | Hashes the random sequence of bytes using keccak512. |
```
Comment (bucanero) [inline docs/2.develop/contracts/environment/table-func.rs.md:7]: `random`

File: docs/2.develop/contracts/environment/table-func.as.md (hunk 1 above)
Comment (idea404) [reply docs/2.develop/contracts/environment/table-func.as.md]: Agreed I've removed random from all three tables

**Review by bucanero — APPROVED** [review]

**Pull request merged**